sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DeadStores.cs

```diff
@@ -869,9 +869,13 @@ public class TupleReturn
     {
         public static (int foo, int bar) M(string text)
         {
-            int b = int.Parse(text); // Noncompliant -- false positive
+            int b = int.Parse(text);
             return (1, b);
         }
-    }
 
+        public void UnusedTuple_FalseNegative()
+        {
+            (int x, int y) t = (1, 2); // Compliant - FN, tuples are not yet covered: https://github.com/SonarSource/sonar-dotnet/issues/2933
```

Comment:
probably it's worth mentioning in the ticket that once that ticket for CSharpExplodedGraph will be handled, this workaround should be removed